fe/pages/dashboards.py

```diff
@@ -1,6 +1,9 @@
+import os
```

Comment:
It seems the code doesn't currently follow the Black code style, could you please set up pre-commit ([instructions](https://github.com/EMBL-EBI-ABC/PerturbationCatalogue?tab=readme-ov-file#set-up)) and run `pre-commit run --all-files`?